source/puppet-agent/_agent1.x.html

```diff
@@ -6,14 +6,15 @@
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#126">1.2.6</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-422">4.2.2</a></td> <td><a href="/facter/3.1/release_notes.html#facter-310">3.1.0</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-303">3.0.3</a></td> <td><a href="/mcollective/releasenotes.html">2.8.6</a></td> <td>2.1.6</td> <td>1.0.2d</td></tr>
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#125">1.2.5</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-422">4.2.2</a></td> <td><a href="/facter/3.1/release_notes.html#facter-310">3.1.0</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-303">3.0.3</a></td> <td><a href="/mcollective/releasenotes.html">2.8.6</a></td> <td>2.1.6</td> <td>1.0.2d</td></tr>
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#124">1.2.4</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-422">4.2.2</a></td> <td><a href="/facter/3.1/release_notes.html#facter-310">3.1.0</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-303">3.0.3</a></td> <td><a href="/mcollective/releasenotes.html">2.8.5</a></td> <td>2.1.6</td> <td>1.0.2d</td></tr>
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#123">1.2.3</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-421">4.2.1</a></td> <td><a href="/facter/3.0/release_notes.html#facter-302">3.0.2</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-301">3.0.1</a></td> <td><a href="/mcollective/releasenotes.html">2.8.3</a></td> <td>2.1.6</td> <td>1.0.0s</td></tr>
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#122">1.2.2</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-421">4.2.1</a></td> <td><a href="/facter/3.0/release_notes.html#facter-302">3.0.2</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-301">3.0.1</a></td> <td><a href="/mcollective/releasenotes.html">2.8.2</a></td> <td>2.1.6</td> <td>1.0.0s</td></tr>
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#121">1.2.1</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-420">4.2.0</a></td> <td><a href="/facter/3.0/release_notes.html#facter-301">3.0.1</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-301">3.0.1</a></td> <td><a href="/mcollective/releasenotes.html">2.8.2</a></td> <td>2.1.6</td> <td>1.0.0s</td></tr>
+    <tr><td><a href="/puppet/4.2/reference/release_notes_agent.html#120">1.2.0</a></td> <td><a href="/puppet/4.2/reference/release_notes.html#puppet-420">4.2.0</a></td> <td><a href="/facter/3.0/release_notes.html#facter-300">3.0.0</a></td> <td><a href="/hiera/3.0/release_notes.html#hiera-301">3.0.1</a></td> <td><a href="/mcollective/releasenotes.html">2.8.2</a></td> <td>2.1.6</td> <td>1.0.0s</td></tr>
```

Comment:
Hmm. Did you edit these by hand? The other ones are automated, and the hand-crafted ones will get zapped the next time we generate the table. 

The mapping to puppet versions will make them a bit trickier to automate, but... no wait, I think I already see how to do it.